src/test/java/duke/task/TodoTest.java

```diff
@@ -0,0 +1,23 @@
+package duke.task;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+public class TodoTest {
+
+    @Test
+    public void toStringTest() {
+        //Correct String representation of the Todo.
+        Todo todo = new Todo("go for run");
+        assertEquals("[T][ ] go for run", todo.toString());
+    }
+
+    @Test
+    public void markDoneTest() {
```

Comment:
Perhaps this should be testMarkDone() so that the naming for tests is more standardized?

Reply:
Yes i agree with your suggestion, thank you!